pages/apprenticeship.js

```diff
@@ -2,17 +2,79 @@ import Page from '../components/base_page';
+                Software Apprenticeship
             </h1>
 
-            <div className={css.grid}>
-                Blah blah blah.
+            <br />
+        
+            <div className={css.card}>
+
+                <h3>What is this?</h3>
+                <p>Our founder, Jonathan Tsai, started an apprenticeship program at Awe & Reverence (A&R) to train aspiring Christian software engineers to equip them with tangible skills to succeed in their craft. Jonathan has mentored dozens of engineers over the years, and through the software apprenticeship has provided a post-bootcamp onbaording experience for more junior engineers to get real world, collaborative, team experience. Several of them have gone on to full-time software engineering roles and are junior, mid-level, and senior-level engineers at household-name tech companies.</p>
+
+                <br />
+
+                <p>Not only will you gain invaluable experience, you'll also be paid for your work! Each feature/task will have a bounty with a dollar amount associated with it. The larger the feature/task, the larger the bounty. We don't believe in muzzling the ox here.</p>
+
+            </div>
+
+            <br />
+
+            <div className={css.card}>
+
+                <h3>What are the qualifications?</h3>
+                <p>Born again Christian that is FAT!</p>
+
+                <br />
+
+                <p>F - Faithful</p>
+                <p>A - Available</p>
+                <p>T - Teachable</p>
+
             </div>
+
+            <br />
+
+            <div className={css.card}>
+
+                <h3>What are the responsibilities?</h3>
+                <p>The first project that A&R has been focused on is called Maskil; envisioned to be the FIRST EVER digital hymnal and worship study Bible. (Imagine every worship song, every language, with (Rap)Genius-style inline annotations and two-way links to Bible passages). Currently Maskil has 350+ songs in English, 650+ songs in Korean, and growing!</p>
+
+                <br />
+
+                <p>We have a feature roadmap planned out for years. You will be working collaboratively with a small team of engineers.</p>
+
+                <br />
+
+                <p>Some of the technologies you will learn and be exposed to include:</p>
+
+                <br />
+
+                <p>-Django</p>
+                <p>-React</p>
+                <p>-MySQL</p>
+                <p>-LESS CSS</p>
+                <p>-Git Workflow</p>
+
+            </div>
+
+            <br />
+        
+            <div className={css.card}>
+                <h3>Say Hello</h3>
+                <p>Interested in joining our software apprenticeship? We'd love to hear from you! Send an email on why you'll be a good fit for this apprenticeship to <b>apprenticeship@aweandreverence.com</b></p>
+
+                <br />
+
+                <Button href="mailto:apprenticeship@aweandreverence.com" variant="outline-dark">Contact Us</Button>
```

Comment:
Could you look into using https://reactjsexample.com/an-intelligent-react-component-to-obfuscate-any-contact-link/ / https://www.npmjs.com/package/react-obfuscate to obfuscate the email address, so that web crawlers on the internet don't crawl our website and then spam our email?

At the minimum, change it to  hello @ aweandreverence dot com (the email) (I don't believe we have set up the former, and don't want to set up an additional email)

Also, since you have it as a `mailto:`, could you set the default `subject` to `Apprenticeship`?